.github/workflows/ci.yml

```diff
@@ -114,7 +115,43 @@ jobs:
       # Ensure all downstream packlet versions are consitent with the base packlet version
       - name: Ensure all package versions are consistent
         run: rush ensure-consistent-versions
-      # Tests
+      # Upload azure-communication api.md files for easy access
+      - name: Upload communication-react api files
+        uses: actions/upload-artifact@v3
+        with:
+          name: communication-react.${{ matrix.flavor }}.api.json
+          path: |
+            packages/communication-react/temp/communication-react.api.json
+
+  jest-test-coverage:
+    needs: [get_matrix, build_packages]
+    name: Jest Test Coverage
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: ${{ fromJSON(needs.get_matrix.outputs.matrix) }}
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE so job can access it
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      # Ensure node version is great enough
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
+      # Try get node_modules from cache
+      - name: Restore node_modules from cache
+        uses: actions/cache@v3
+        with:
+          path: common/temp/pnpm-store
+          key: ${{ runner.os }}-${{ hashFiles('common/config/rush/pnpm-lock.yaml') }}
+      # Install dependencies
+      - name: Install rush
+        run: npm install -g @microsoft/rush
+      - name: Install dependencies
+        run: rush install --max-install-attempts 3
+      - name: Build Packages
+        run: rush build -t @azure/communication-react
```

Comment:
just a thought but can we reuse result of build command instead of building every time?

Reply:
Maybe; it's complicated with the number of things that get built in each packlet. There is a cache action, but I was not able to get it happy in the time box I had.